ament_copyright/ament_copyright/licenses.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# to add more licenses do not extend this file
+# instead create a separate package and register custom licenses as entry points
+
+from collections import namedtuple
+
+LicenseEntryPoint = namedtuple(
+    'LicenseEntryPoint', ['name', 'file_header', 'license_file', 'contributing_file'])
+
+apache2 = LicenseEntryPoint(
+    'Apache License, Version 2.0',
+
+    """\
+{copyright}
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.""",
+
+    """\
```

Comment:
Wouldn't it be better to move the text of the license to a separate file and read it?